### DISCUSSION TITLE
Support for streaming new database entries via WATCH API

### DISCUSSION THREAD

**Discussion by fionera** [grafana/clickhouse-datasource#292] [Ideas]
Clickhouse supports streaming new elements in a Table via the WATCH statement. It would be cool to have support for this via the grafana streaming datasource api

https://clickhouse.com/docs/en/sql-reference/statements/watch
https://grafana.com/docs/grafana/latest/developers/plugins/build-a-streaming-data-source-plugin/

**Comment by SpencerTorres** [comment]
Tracked in https://github.com/grafana/clickhouse-datasource/issues/183.

`WATCH` statement is experimental, and `LIVE VIEW` is deprecated. This is something I'm interested in trying out for logs
